src/Bedrock.Framework.Experimental/Protocols/Tls/TlsFrameType.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Bedrock.Framework.Experimental.Protocols.Tls
+{
+    internal enum TlsFrameType : byte
```

Comment:
Link to spec?

Reply:
Linked to spec and changed to "TlsRecordType" rather than frame to match the TLS RFC Spec